.github/workflows/tex-pdf.yml

```diff
@@ -55,16 +55,15 @@ jobs:
           mkdir -p datasets/oneoff/pdf
           for changed_file in ${{ steps.files.outputs.added_modified }}; do
             if [[ "$changed_file" == **.cgel ]]; then
-              # make .tex
               filename=$(basename "$changed_file")
               tree_name="${filename%.cgel}"
-              python cgel/tree2tex.py ${changed_file} > datasets/oneoff/tex/$tree_name.tex
-              git add datasets/oneoff/tex/$tree_name.tex
-              git commit -m "generated tex file for $filename"
+              # make .tex
+              python cgel/tree2tex.py "${changed_file}" > datasets/oneoff/tex/"$tree_name".tex
```

Comment:
I think the quotes have to be around what will become a single string, not around just the variable

Reply:
I think that for filepaths it's sufficient to quote the variable:

```
$ test="this is a test"
$ cd ~/"$test"
-bash: cd: /Users/bw/this is a test: No such file or directory
```

Reply:
Interesting:

```bash
$ test="blah blah blah"
$ $ python -c "import sys; print(sys.argv)" x"$test"y$test
['-c', 'xblah blah blahyblah', 'blah', 'blah']
```

Looks like the scope of the quotes doesn't really matter as long as it contains any spaces + variables whose values contain spaces